source/print.cpp

```diff
@@ -27,6 +27,8 @@
 #include "print.h"
 
 #if defined(SPIRV_LINUX) || defined(SPIRV_MAC)
+namespace libspirv {
```

Comment:
I thought about that while coding it.  I didn't want to have a namespace libspirv around the #include of <Windows.h>  So either we have to declare the libspirv namespace twice, or we have to do the conditional compilation of defined(SPIRV_WINDOWS) twice.
Sorry I didn't reply to this earlier.